kpi/views/v2/data.py

```diff
@@ -460,9 +481,9 @@ def _filter_mongo_query(self, request):
         # submissions at one time
         limit = filters.get('limit', settings.SUBMISSION_LIST_LIMIT)
         try:
-            filters['limit'] = positive_int(limit,
-                                            strict=True,
-                                            cutoff=settings.SUBMISSION_LIST_LIMIT)
+            filters['limit'] = positive_int(
```

Comment:
\*gasp\* is that a private method? :open_mouth: :zany_face: 
```python
from rest_framework.pagination import _positive_int as positive_int
```

Reply:
you pulled up the definition for me, and it seems totally safe to use; i'm not sure why DRF elected to make it private:
```python
def _positive_int(integer_string, strict=False, cutoff=None):
    """
    Cast a string to a strictly positive integer.
    """
    ret = int(integer_string)
    if ret < 0 or (ret == 0 and strict):
        raise ValueError()
    if cutoff:
        return min(ret, cutoff)
    return ret
```
i guess this is a good example of why we don't always have to be dogmatic about following rules (guidelines?) around these so-called "weak 'internal use' indicators"